apps/console/src/features/groups/api/groups.ts

```diff
@@ -117,6 +117,28 @@ export const getGroupById = (groupId: string): Promise<any> => {
     });
 };
 
+/**
+ * Retrieve Group details for a given group id excluding members.
+ *
+ * @param groupId group id to retrieve group details
+ */
+export const getGroupByIdExcludingMembers = (groupId: string): Promise<any> => {
```

Comment:
Can we avoid `any` here. What's the type of the response?

```suggestion
export const getGroupByIdExcludingMembers = (groupId: string): Promise<any> => {
```

Reply:
Fixed with 7ce782bb9607f14623bf58fab36cf175bf55ae2f